### PR TITLE
Shop Model Test

### PR DESCRIPTION
**Pull request by LevinGeorge**
@danielpaul Need to clarify 2 things with @IonatanSala before merging.

### CODE REVIEW

File: spec/models/shop_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      should validate_length_of(:description).is_at_least(10).on(:create)
+    end
+  end
+
+  describe "Association Test" do
+    it { should belong_to(:user) }
+    it { should have_many(:withdrawals) }
+    it { should have_many(:products).dependent(:delete_all) }
+
+    # it "deletes dependent products" do
```
Comment (IonatanSala) [inline spec/models/shop_spec.rb]: @LevinGeorge Can you remove commented out code that isn't needed, as Daniel always tells me :P

File: spec/models/shop_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    #   shop = build(:shop)
```
Comment (IonatanSala) [inline spec/models/shop_spec.rb]: @LevinGeorge Here too :P

**Comment by danielpaul** [comment]
@LevinGeorge @IonatanSala could you guys have a look at this and see what still need to be done and get this across the line? I'd like to merge the testing branch before I work on further UI as that could break things.

File: spec/models/shop_spec.rb (hunk 1 above)
Comment (LevinGeorge) [inline spec/models/shop_spec.rb]: :+1:

**Comment by LevinGeorge** [comment]
@danielpaul Ready to merge :+1:

**Pull request merged**